tensorboard/plugins/interactive_inference/README.md

```diff
@@ -227,3 +230,14 @@ We imagine WIT to be useful for a wide variety of users.
   model on a dataset. Try it out with your own data.
 * Lay users - Learn about machine learning by interactively playing with
   datasets and models.
+
+## How do I use it in a Jupyter notebook?
+First, install and enable WIT for Jupyter through the following commands:
+```sh
+pip install witwidget
+jupyter nbextension install --py --symlink --sys-prefix witwidget
```

Comment:
I know the cookiecutter example has this but do we know why `sys-prefix` is correct? What does it do if you are using virtualenv?

Reply:
sys-prefix works appropriately with virtualenv and conda (https://jupyter-notebook.readthedocs.io/en/4.x/examples/Notebook/rstversions/Distributing%20Jupyter%20Extensions%20as%20Python%20Packages.html)